recipes/python-jenkins/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "python-jenkins" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 62114d830cc95dee9d9a75f5c4e0fd2ea7b0ca17b0f09357a01a22432c09d325
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pbr >=0.8.2
+    - pip
+    - python
+  run:
+    - multi_key_dict
+    - python
+    - requests
+    - setuptools  # requires pkg_resources at runtime
+    - six >=1.3.0
+
+test:
+  imports:
+    - jenkins
+
+about:
+  home: http://git.openstack.org/cgit/openstack/python-jenkins
+  license: BSD
```

Comment:
Version?

Reply:
Looks like it is BSD-3-Clause: http://git.openstack.org/cgit/openstack/python-jenkins/tree/COPYING